Improbable/Directory.Build.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is a workflow improvement for local iteration that I accidentally checked in

Reply:
(still WIP on the consumer side of things), this part is complete